src/components/v2/appDetails/sourceInfo/environmentStatus/EnvironmentStatus.component.tsx

```diff
@@ -16,6 +16,7 @@ function EnvironmentStatusComponent({appStreamData}:{appStreamData: any}) {
     const [appDetails] = useSharedState(IndexStore.getAppDetails(), IndexStore.getAppDetailsObservable());
     const [showAppStatusDetail, setShowAppStatusDetail] = useState(false);
     const status = appDetails.resourceTree?.status || '';
+    const isHibernated = status.toLowerCase().includes('hibernated')
```

Comment:
includes or equals?

Reply:
It'll work with `includes` as we have `hibernated` in both the statuses and will handle both scenarios. However, as discussed, I've put the specific checks just to make sure we're showing it only for those specific statuses. :)